### PR TITLE
@mobile version styled + { css } in person tile + tails height

### PR DESCRIPTION
**Pull request by Gosia-Magdzik**
.... as in the title

### CODE REVIEW

**Review by marta-zebrowska — APPROVED** [review]
Good work Gosia!!! :)

File: src/features/movies/moviesPage/BackgroundPoster/index.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React from "react";
-import { Rating, Star } from "../../../../common/Rating";
-import { Wrapper, PosterSpace, Poster, Section, Title, RateStyledA, RateStyledB, Votes } from "./styled";
+import { Rating } from "../../../../common/Rating";
+import { Wrapper, PosterSpace, Poster, Section, Title, RateStyledA, RateStyledB, Votes, Star } from "./styled";
```
Comment (MarcinkPawel) [inline src/features/movies/moviesPage/BackgroundPoster/index.js:3]: that is great solution for the Star, I  wanted to make it way diffrent. but yours is simpler :)

File: src/features/people/PersonDetails/AboutPerson/index.js [hunk 2]
```diff
@@ -26,14 +27,19 @@ export const AboutPerson = ({
...
+            <Info>{new Date(birthday).toLocaleDateString()}</Info>
+          </DetailsLine>
+          <DetailsLine constant>
+            {birthplace ? (
+              <>
+                <Label constant>Place of birth:</Label>
+                <Info> {birthplace} </Info>{" "}
+              </>
+            ) : null};
+          </DetailsLine>
```
Comment (MarcinkPawel) [inline src/features/people/PersonDetails/AboutPerson/index.js:42]: great solution!!!

File: src/features/people/PersonDetails/AboutPerson/styled.js [hunk 3]
```diff
@@ -85,11 +87,24 @@ export const Label = styled.p`
...
+    display: none;
   }
+
+  ${({ constant }) => constant && css`
+    @media (max-width: ${({ theme }) => theme.breakpoint.mobileMax}px) {
+      display: block;
+      font-size: 12px;
+    }
+  `}
+
```
Comment (MarcinkPawel) [inline src/features/people/PersonDetails/AboutPerson/styled.js:102]: I had similar idea for the star in rating

**Review by MarcinkPawel — APPROVED** [review]
Nice job!

**Pull request merged**